.github/workflows/ZygoteCI.yml

```
@@ -0,0 +1,30 @@
+name: ZygoteCI
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+
+jobs:
+  test:
+    name: Julia v${{ matrix.julia-version }} - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.0, 1.4]
```

Lets just do:
```suggestion
        julia-version: [1]
```

I think Zygote is dropping support for the LTS soon and I think 1 gets the latest stable release?